mod/rng-validate/src/main/com/thaiopensource/relaxng/util/resources/Messages.properties

```diff
@@ -11,6 +11,9 @@ resolver_not_found=Use of catalogs requires Apache XML Commons Resolver v1.2.\n\
   Then put resolver.jar in the same directory as jing.jar.
 
 usage=Jing version {0}\n\
-usage: java com.thaiopensource.relaxng.util.Driver [-i] [-c] [-s] [-t] [-C catalogFile] [-e encoding] RNGFile XMLFile...\n\
+\n\
+USAGE: java com.thaiopensource.relaxng.util.Driver [-i] [-c] [-s] [-t] [-x] [-C catalogFile] [-S svrlFile] [-e encoding] RNGFile XMLFile...\n\
```

Comment:
explain `-S svrlFile` on a line of its own: that it's a SVRL error report compatible with Schematron, which enables  unified error handling for both schema errors and Schematron errors.
Put similar comment in the branch description and PR, since "EHRI-nnn" doesn't tell anything to people outside our project (who don't have access to our jira either)